src/backend/image/ResolveID.js

```diff
@@ -1,2 +1,35 @@
 let cover = 1;
```

Comment:
I don't really get that. So you are exporting `1`, why do you need the `coverObj`?